packages/hoppscotch-backend/src/user/user.service.ts

```diff
@@ -114,14 +114,15 @@ export class UserService {
    * @param userUid User uid
    * @returns Either of User with updated refreshToken
    */
-  async UpdateUserRefreshToken(refreshTokenHash: string, userUid: string) {
+  async updateUserRefreshToken(refreshTokenHash: string, userUid: string) {
     try {
       const user = await this.prisma.user.update({
         where: {
           uid: userUid,
         },
         data: {
           refreshToken: refreshTokenHash,
+          lastLoggedOn: new Date(),
```

Comment:
Here too, the user isnt techincally logged-in when the refresh tokens are updated.

Reply:
Also are we sure the date is in the correct format expected by the database.

Reply:
The date format is the same as `createdOn` perspective of DB. So, it's good.